sat/cli/bootprep/main.py

```diff
@@ -324,6 +324,7 @@ def print_report(args, instance, created_images):
             item_class = IMSInputImage
         else:
             created = items.created
+            skipped = items.skipped_items
```

Comment:
Unused variable.

Reply:
Doh, I was not supposed to have committed that!